src/AntTweakBar/TwMgr.cpp

```diff
@@ -6352,6 +6352,8 @@ void CTwMgr::SetCursor(CTwMgr::CCursor _Cursor)
         if( ok && (ci.flags & CURSOR_SHOWING) )
             ::SetCursor(_Cursor);
     }
+#else
+	(void)_Cursor;
```

Comment:
The advantage of this technique compared to pragmas is that it works accross all compilers and I've actually seen this quite often in other c code bases.